src/AblyRest.php

```diff
@@ -18,6 +18,33 @@ class AblyRest {
     public $options;
     protected static $libFlavour = '';
 
+    /**
+     * Map of agents that will be appended to the agent header.
+     *
+     * This should only be used by Ably-authored SDKs.
+     * If you need to use this then you have to add the agent to the agents.json file:
+     * https://github.com/ably/ably-common/blob/main/protocol/agents.json
+     * The keys represent agent names and its corresponding values represent agent versions.
+     */
+    protected static $agents = array();
+
+    static function ablyAgentHeader()
+    {
+        $sdk_identifier = 'ably-php/'.self::LIB_VERSION;
+        $runtime_identifier = 'php/'.Miscellaneous::getNumeric(phpversion());
+        $agent_header = $sdk_identifier.' '.$runtime_identifier;
+        // TODO -  remove $libFlavour check once deprecated method setLibraryFlavourString is deleted
+        if (self::$libFlavour == 'laravel') {
```

Comment:
How about we update `setLibraryFlavourString` to just call `setAblyAgentHeader($flavour)`? Then we don't need `$libFlavour` anymore

Reply:
Done

Reply:
Great 👍